src/App.js

```diff
@@ -1,25 +1,74 @@
 import React from 'react';
+import { Clock } from './components/Clock/Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    name: 0,
+  };
 
+  setRandomName = () => {
+    const oldName = this.state.name;
+    const random = Math.floor(Math.random() * 100);
+
+    this.setState({
+      name: random,
+    });
     // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+    console.log(`The Clock was renamed from ${oldName} to ${random}`);
+  }
+
+  showClock() {
```

Comment:
Думаю, так було би краще
```suggestion
  showClock = () => {
```